src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -289,12 +298,13 @@ class ActivityFeed extends React.Component<Props, State> {
 
         return (
             // eslint-disable-next-line
-            <div className="bcs-activity-feed" data-testid="activityfeed" onKeyDown={this.onKeyDown}>
+            <div className={classNames("bcs-activity-feed", { "bcs-header-shadow": this.state.isHeaderShadowVisible })} data-testid="activityfeed" onKeyDown={this.onKeyDown}>
```

Comment:
could we use SUIT naming for new classes?

I guess this would be considered a state? https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md

Reply:
I see we've used `is-scrolled[...]` before in this repo so `bcs-is-scrolled` seems like an option

Reply:
Great suggestion. Will do that.